crates/optimism/rpc/src/sequencer.rs

```diff
@@ -104,34 +83,26 @@ impl SequencerClient {
         condition: TransactionConditional,
     ) -> Result<(), SequencerClientError> {
         let params = json!([format!("0x{}", hex::encode(tx)), condition]);
-        let body =
-            self.request_body("eth_sendRawTransactionConditional", params).map_err(|_| {
+
+        self.send_rpc_call("eth_sendRawTransactionConditional", params).await.inspect_err(
```

Comment:
here we then can alos use a tuple (encoded,condition)

see also https://github.com/alloy-rs/alloy/blob/72a12eab59df688abce53da845ee66dcc1060c86/crates/provider/src/provider/trait.rs#L876-L876

Reply:
Done :)